lib/ansible/modules/files/find.py

```diff
@@ -369,6 +370,16 @@ def main():
 
     params = module.params
 
+    # Set the default match pattern to either a match-all glob or
+    # regex depending on use_regex being set.  This makes sure if you
+    # set excludes: without a pattern pfilter gets something it can
+    # handle.
+    if not params['patterns']:
+        if params['use_regex']:
+            params['patterns'] = '.*'
```

Comment:
Can you set this as a list instead of a string. The module option has `type='list'` which means it is expecting a list instead of a string.

Reply:
Done